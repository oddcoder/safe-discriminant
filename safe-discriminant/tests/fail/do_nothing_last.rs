use safe_discriminant::Discriminant;
use safe_discriminant_derive::do_nothing;

#[repr(u8)]
#[derive(Discriminant)]
#[do_nothing]
pub enum Foo {
    A = 0,
    B = 1,
}

fn main() {}
