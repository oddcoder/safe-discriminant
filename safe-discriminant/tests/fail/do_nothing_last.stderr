error: Discriminant is not compatiable with any top level `#[attr]` except `#[repr(_)]`.
 --> tests/fail/do_nothing_last.rs:7:10
  |
7 | pub enum Foo {
  |          ^^^
